NextcloudTalk/NCRoomParticipants.m

```diff
@@ -72,4 +74,22 @@ - (NSString *)displayName
     }
     return _displayName;
 }
+
+- (NSString *)callIconImageName
+{
+    if (self.inCall == CallFlagDisconnected) {
```

Comment:
Maybe we should rename `inCall` to `callFlag` to avoid confusion? Maybe better add a `isInCall` as boolean with a getter if we need it?

Reply:
`inCall` is the name of the attribute that comes from the server (when fetching participants). Naming it `callFlag` would make sense but I think it could created confusion :P

https://github.com/nextcloud/talk-ios/blob/master/NextcloudTalk/NCRoomParticipants.m#L37

Reply:
Oh, sorry, I thought in the backend it was callFlags, but it’s only called callFlag for conversations...

Reply:
No problem ;) we kept the attribute name for legacy reasons. In the past it was a just boolen variable but then it was extend to include callFlags.